Arm/source/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include "RoverArmSystem.hpp"
+
+int main()
+{
+  sjsu::RoverArmSystem armControl;
+  armControl.Initialize();
+  armControl.Home();
+  while (true)
+  {
+    armControl.Get_data();
+    armControl.Move_arm();
```

Comment:
This NEEDS a delay. 
```suggestion
    armControl.Move_arm();
    sjsu::Delay(100ms);
```

Reply:
Attempting to read data from the server as quickly as possible without any sort of delay will cause the server to be overloaded which is basically a DDOS on our own systems and will result in undefined behavior.